include/netdb.h

```diff
@@ -21,6 +21,39 @@
 #include <stdint.h>
 
 
+#define AI_PASSIVE     1 << 0
```

Comment:
if there is an expression in macro define, it needs to be inside parentheses `(1 << 0)` to avoid possible problems in the future. See https://godbolt.org/z/fbM539zEW for quick sample why this is bad :).

Reply:
Oh, that's right, I've missed it, sorry. It's fixed now.